### DISCUSSION TITLE
what are the hp drain values

### DISCUSSION THREAD

**Discussion by 1vers1on** [ppy/osu#18250] [Q&A] [answered]
What are the drain values for hp0-10

also how much hp does each note give?

**Comment by bdach** [comment, marked as answer]
drain:

https://github.com/ppy/osu/blob/bece807857be50d46e50e98a38b3bb88585a0d0f/osu.Game/Rulesets/Scoring/DrainingHealthProcessor.cs#L16-L18

health increases:

https://github.com/ppy/osu/blob/8edbc10155ad3591889f0fb822a9d63ae07c455b/osu.Game/Rulesets/Judgements/Judgement.cs#L78-L126
https://github.com/ppy/osu/wiki/Scoring#hit-results
